src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -732,28 +746,12 @@ private bool TryScanInterpolatedString(ref TokenInfo info)
         {
             Debug.Assert(TextWindow.PeekChar() == '$');
 
-            if (TextWindow.PeekChar(1) == '$')
+            if (TextWindow.PeekChar(1) is '$' or '@' or '"')
             {
                 // $$ - definitely starts a raw interpolated string.
```

Comment:
Probably should add the comment about error cases that was deleted below?

Reply:
indeed.